source/main/windows/main.js

```diff
@@ -46,7 +47,9 @@ export const createMainWindow = () => {
   window.loadURL(`file://${__dirname}/../renderer/index.html`);
   window.on('page-title-updated', event => { event.preventDefault(); });
 
-  let title = `Daedalus (${environment.version}#${environment.build})`;
+  const { version, build, API_VERSION } = environment;
+  const buildNumber = uniq([API_VERSION, build]).join('.');
+  let title = `Daedalus (${version}#${buildNumber})`;
```

Comment:
Wouldn't it be more advantageous to have this built in a separated file, so it would be easier to reuse it (like at `paperWalletPdfGenerator.js`) and make future amendments?